src/main/scala/uk/gov/nationalarchives/tdr/api/auth/ValidationAuthoriser.scala

```diff
@@ -25,13 +25,13 @@ class ValidationAuthoriser(implicit executionContext: ExecutionContext)
     // All fields must have an authorisation tag defined. This means that if we forget to add authorisation, the
     // query is blocked by default, which prevents some security bugs.
     val isTopLevelField = ctx.path.path.length == 1
-    val fieldHasAuthTag = ctx.field.tags.exists(tag => tag.isInstanceOf[AuthorisationTag])
+    val fieldHasAuthTag = ctx.field.tags.exists(tag => tag.isInstanceOf[ValidationTag])
     if (isTopLevelField && !fieldHasAuthTag) {
       throw new AssertionError(s"Query '${ctx.field.name}' does not have any authorisation steps defined")
     }
 
     val validationList: Seq[BeforeFieldResult[ConsignmentApiContext, Unit]] = ctx.field.tags.map {
-      case v: AuthorisationTag => {
+      case v: ValidationTag => {
```

Comment:
Is this check strict enough now? The goal of this check was to make sure every API endpoint had some authorisation defined. It's not enough for it to have state validation but not authorisation.

Reply:
Reverted back to authorisation tag type, and implemented separate sangria middleware to check consignment state validation as discussed